src/python/model_api/models/result/segmentation.py

```diff
@@ -149,13 +149,18 @@ def rotated_rects(self, value):
 
 
 class Contour:
-    def __init__(self, label: str, probability: float, shape: list[tuple[int, int]]):
+    def __init__(self, label: str, probability: float, shape: np.ndarray, child_shapes: list[np.ndarray] | None = None):
```

Comment:
Suggestion: to avoid an API-breaking changes, allow both list and np.ndarray (`shape: list[tuple[int, int] | np.ndarray`) and convert from one to the other in the constructor. Also, document the expected structure of the lists/arrays in the docstring.